merge_reports.js

```diff
@@ -31,7 +31,10 @@ const mergeSummaries = (summaries) => {
  * for object schema see: https://developer.github.com/v3/checks/runs/#output-object
  */
 module.exports = (reports) => {
-  const annotations = reports.map(r => r.annotations).flat()
+  let annotations = []
```

Comment:
This change is unrelated and should not be part of this PR.

Reply:
This change was merged as part of PR #247